mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -2107,18 +2107,14 @@ def LLVM_CallIntrinsicOp
                        VariadicOfVariadic<LLVM_Type,
                                           "op_bundle_sizes">:$op_bundle_operands,
                        DenseI32ArrayAttr:$op_bundle_sizes,
-                       OptionalAttr<ArrayAttr>:$op_bundle_tags);
+                       OptionalAttr<ArrayAttr>:$op_bundle_tags,
+                       OptionalAttr<DictArrayAttr>:$arg_attrs,
+                       OptionalAttr<DictArrayAttr>:$res_attrs);
   let results = (outs Optional<LLVM_Type>:$results);
   let llvmBuilder = [{
     return convertCallLLVMIntrinsicOp(op, builder, moduleTranslation);
   }];
-  let assemblyFormat = [{
-    $intrin `(` $args `)`
-    ( custom<OpBundles>($op_bundle_operands, type($op_bundle_operands),
-                        $op_bundle_tags)^ )?
-    `:` functional-type($args, $results)
-    attr-dict
-  }];
+  let hasCustomAssemblyFormat = 1;
```

Comment:
We probably also want to add a verifier that calls `verifyOperandBundles` as it is done for the call?

Reply:
Ideally we can also duplicate a test like this one https://github.com/llvm/llvm-project/blob/4fb31e44017bd9ad914fc4c814fbaa60cc18d5b4/mlir/test/Dialect/LLVMIR/invalid.mlir#L1687 to test it actually executes. I assume the logic would be identical.

Reply:
>  add a verifier that calls verifyOperandBundles as it is done for the call?

I didn't change previous behavior, seems like that was already missing for `llvm.call_intrisinc`, if no test fails I'll add that to this PR.

Reply:
Looks like we already verify it:
```
LogicalResult CallIntrinsicOp::verify() {
  if (!getIntrin().starts_with("llvm."))
    return emitOpError() << "intrinsic name must start with 'llvm.'";
  if (failed(verifyOperandBundles(*this)))
    return failure();
  return success();
}
```
Just added tests.